examples/call-record-aggregator/project/cloudflow-plugins.sbt

```diff
@@ -2,5 +2,5 @@
 //
 resolvers += "Akka Snapshots" at "https://repo.akka.io/snapshots/"
 resolvers += Resolver.url("cloudflow", url("https://lightbend.bintray.com/cloudflow"))(Resolver.ivyStylePatterns)
-addSbtPlugin("com.lightbend.cloudflow" % "sbt-cloudflow" % "1.3.1")
+addSbtPlugin("com.lightbend.cloudflow" % "sbt-cloudflow" % "1.3.2-SNAPSHOT")
```

Comment:
This is not a published build, right. AFAIk you need to create an internal release to make this work.

Reply:
Yet the CI build is not broken for reasons I do not understand unless someone has pushed a SNAPSHOT jar to the repo.

Reply:
ah right. It is because I run the CI tests. I'll revert those changes.

Reply:
The CI build a `-SNAPSHOP` version of the core locally, and update all the examples project to use this built version. So it doesn't really care of this version string.
It is made to ensure that all the examples are up-to-date with the core code.

Reply:
True, but it would break the build for anyone else checking out this branch, or master after merging this, since they wouldn't have the SNAPSHOT build (or they have an older one)